_topic_map.yml

```diff
@@ -1473,7 +1473,7 @@ Topics:
     File: planning-migration-3-to-4
   - Name: Migration tools and prerequisites
     File: migrating-application-workloads-3-4
-  - Name: Deploying the Migration Toolkit for Containers
+  - Name: Deploying and upgrading the Migration Toolkit for Containers
```

Comment:
Deploying and upgrading need to be separate assemblies because they're separate user stories. If you have a list of improvement ideas, please add this to it.